deployment/deployment-grpc-v2/02-contour.yaml

```diff
@@ -49,6 +49,10 @@ spec:
         volumeMounts:
         - name: contour-config
           mountPath: /config
+        lifecycle:
+          preStop:
+            exec:
+              command: ["wget", "-qO-", "http://localhost:9001/healthcheck/fail"] 
```

Comment:
isn't this supposed to be a [POST](https://www.envoyproxy.io/docs/envoy/v1.9.0/operations/admin#post--healthcheck-fail)?

Reply:
I've confirmed this doesn't work, it returns a 400 error.

Adding `--post-data=''` is the fix.